include/deal.II/grid/manifold.h

```diff
@@ -1202,8 +1202,8 @@ namespace Manifolds
 
         if (with_laplace)
           {
-            std::fill(points_weights.second.begin(), points_weights.second.begin()+4, 1.0/16.0);
-            std::fill(points_weights.second.begin()+4, points_weights.second.end(), 3.0/16.0);
+            std::fill(points_weights.second.begin(), points_weights.second.begin()+4, -0.25);
+            std::fill(points_weights.second.begin()+4, points_weights.second.end(), 0.5);
```

Comment:
It's now no longer a Laplace smoothing, so the `with_laplace` flag is not the correct name any more. Can you update this in the documentation of the function?

Reply:
I will definitely change this, part of what I considered "boring" in my original post.